### ISSUE TITLE
Time constraints getting lost on 2.2.10

### ISSUE THREAD

**Issue by jwelter1971** [danobot/entity-controller#14]
2.2.10 had been working well for several days now, but interestingly after a few days the time time constraints are no longer respected.  The motion config shows ‘idle’ for status versus ‘constrained’ and is active.   But other motion condors with the same constraints seem to be ok so it’s not all being lost.  A restart fixes it and immediate shows the constrained status

**Comment by danobot** [comment]
Any error logs?
Please update to `2.3.0`

**Comment by jwelter1971** [comment]
Hi Danny - I will get logs tomorrow when back from a business trip.  I only know of the problem as my wife reported it and I got into the UI to see the lightinSM in the idle state versus constrained.  I restarted HA and they went into constrained mode.

I also will update to 2.3.0

One thing it could be is I use constraints to enable from 7am to 10pm; but then also a over-ride based on outside sun elevation and clouds to block if bright out.  Any odd interactions possible between the two?
